### PR TITLE
Final GDPR punchlist

### PR DESCRIPTION
**Pull request by jasonblais**
Final GDPR Punchlist

1. Docs
- [X] Document encryption options
- [x] Update [export compliance overview](https://docs.mattermost.com/overview/compliance.html) to "Certifications and Compliance Overview" and add a section about GDPR.
- [x] Add a section "How is Mattermost licensed" about GDPR and licensing to FAQ https://docs.mattermost.com/overview/faq.html
  - later move this to website and link the question "How is Mattermost licensed?" to that page

2. Website
- [X] Add PDA (data protection addendum) to website --> https://about.mattermost.com/default-data-processing-addendum/
- [x] Update privacy policy with [new legal changes](https://docs.google.com/document/d/1iu800etqkyxf4VUa-V1--M45IvXQbWy9GN9m5rYrVSE/edit) and include reference to DPA
- [x] Include reference to Conditions of User (https://about.mattermost.com/default-terms/) to EE License Agreement: https://about.mattermost.com/enterprise-edition-license/
- [x] Redirect HPNS privacy to new privacy policy with DPA which covers it
- [x] Add to https://www.mattermost.org/licensing/ a link to commercial licenses of Mattermost

3. Server Repo
- [ ] ~~Update privacy policy with DPA (may not need update, or very minor, have to review)~~

4. Legal
- [ ] Complete Privacy Shield self-certification with internal HR privacy policy 
- [x] Review and update other terms and privacy policies (don't believe changes are required]
- [x] Confirm no issues combining E10 and E20 terms, and whether DPA is needed to be referenced
- [x] Ensure Security Alert and Error Reporting and Diagnostics info are properly referenced throughout the documents
- [ ] ~~Update Professional Services Agreement~~
- [x] Update Professional Services Consulting Agreement
- [X] Test privacy@mattermost.com referenced in PDA works

### CODE REVIEW

File: source/overview/compliance.rst [hunk 1]
```diff
@@ -1,6 +1,76 @@
...
+The following overview summarizes how Mattermost, Inc. and Mattermost software comply with European Union's General Data Protection Regulation, also known as Regulation (EU): 2016/679 (`See full text <http://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32016R0679&from=EN>`_)
+
+Continual Commitment to the Principles of GDPR 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost is a collaboration hub for high trust organizations that is committed to supporting the principles of GDPR in protecting the data of people in the European Union through: 
+
+**Security Infrastructure** - Continual investment in security, privacy and compliance capabilities 
+**Contractual Obligations** - Appropriate contractual obligations through our terms of service, including the Data Processing Addendum in our standard **Terms of Service**
+**Privacy Measures** - Self-certification to international standards for data transfer standards including **E.U.-U.S. Privacy Shield**
```
Comment (jasonblais) [inline source/overview/compliance.rst]: need to add link to privacy shield once published

File: source/overview/compliance.rst [hunk 2]
```diff
@@ -1,6 +1,76 @@
...
+
+The following overview summarizes how Mattermost, Inc. and Mattermost software comply with European Union's General Data Protection Regulation, also known as Regulation (EU): 2016/679 (`See full text <http://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32016R0679&from=EN>`_)
+
+Continual Commitment to the Principles of GDPR 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost is a collaboration hub for high trust organizations that is committed to supporting the principles of GDPR in protecting the data of people in the European Union through: 
+
+**Security Infrastructure** - Continual investment in security, privacy and compliance capabilities 
+**Contractual Obligations** - Appropriate contractual obligations through our terms of service, including the Data Processing Addendum in our standard **Terms of Service**
```
Comment (jasonblais) [inline source/overview/compliance.rst]: needs links to DPA and terms

File: source/overview/compliance.rst [hunk 3]
```diff
@@ -1,6 +1,76 @@
...
+**Responsible Disclosure Policy** for security researchers around the world to confidentially report suspected vulnerabilities, which can be addressed in updates to Mattermost software 
+**Security Reviews** conducted by both our own internal security review team and external security researchers. 
+**ISO 27001 Standards** which are met to achieve alignment with international security guidelines. 
+
+Contractual Obligations 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost adheres to contractual obligations for ensuring the proper management of data through: 
+
+**GDPR compliant Data Processing Addendum** included with Mattermost’s standard terms
```
Comment (jasonblais) [inline source/overview/compliance.rst]: Links to DPA and privacy policy

File: source/overview/compliance.rst [hunk 4]
```diff
@@ -1,6 +1,76 @@
...
+**Mattermost Privacy Policy** sharing how data is handled on the online infrastructure controlled by Mattermost, Inc. 
+
+Privacy Measures 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost self-certifies that it adheres to the **E.U.-U.S. Privacy Shield** principles of notice, choice, onward transfer, security, data integrity, access, and enforcement for personal data submitted by our customers and partners.
```
Comment (jasonblais) [inline source/overview/compliance.rst]: add link to privacy shield

**Comment by jasonblais** [comment]
IJT reviewed. 

@amyblais 
1 - Can you help with editor review.
2 - Can you ask someone from security team (Chris/Harrison/George) to review the encryption-options.rst file ONLY by TODAY

File: source/administration/encryption.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
+Encryption Options
+=======================
+
+Mattermost provides encryption-in-transit and encryption-at-rest for both database and file storage. This page guides you through setting up appropriate encryption security.
```
Comment (crspeller) [inline source/administration/encryption.rst]: MM doesn't do encryption at rest. You say this later.

File: source/administration/encryption.rst [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+Encryption is not required for GDPR, although it can be used as an additional safeguard against data breach.
```
Comment (crspeller) [inline source/administration/encryption.rst]: Why are you telling me this?

File: source/administration/encryption.rst [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+.. contents::
+  :backlinks: top
+  :local:
+
+Encryption-in-transit
+-----------------------
+
+Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions between Mattermost client applications and the Mattermost server across both LAN and internet. You may either set up TLS on the Mattermost Server or install a proxy such as NGINX and set up TLS on the proxy. Refer to our `configuration guide for more details <https://docs.mattermost.com/install/config-tls-mattermost.html>`_.
```
Comment (crspeller) [inline source/administration/encryption.rst]: TLS supports many cipher suites: https://en.wikipedia.org/wiki/Transport_Layer_Security#Algorithm
Seems a bit silly to specify one. 

"across both LAN and internet". Is unnecessary specificity.

File: source/administration/encryption.rst [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+Connections to Active Directory/LDAP can `optionally be secured with TLS or stunnel <https://docs.mattermost.com/administration/config-settings.html#id11>`_ in Enterprise Edition E10 and E20.
```
Comment (crspeller) [inline source/administration/encryption.rst]: Sounds like we are limiting the encryption feature, when we are limiting the LDAP feature to EE. Maybe drop the "in Enterprise Edition..."?

File: source/administration/encryption.rst [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+Encryption-at-rest
+-----------------------
+
+Database
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable end user search and compliance reporting of message histories, Mattermost does not offer native encryption-at-rest within the database.
```
Comment (crspeller) [inline source/administration/encryption.rst]: Seems maybe unnecessary to say we don't support native encryption at rest. Why not go right to the solution? Start with  "Encryption-at-rest is available for messages via hardware and software disk encryption solutions applied to the Mattermost database..."

File: source/overview/compliance.rst [hunk 10]
```diff
@@ -1,6 +1,76 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost supports features that ensure data management and data portability.
+
+Data Management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Data retention**: Use `data retention <https://docs.mattermost.com/administration/data-retention.html>`_ to automatically erase data after a set period of time, meeting Right to Erasure principle. In Team Edition, you can use database scripts.
+
+- **Profile deletion**: Delete a person’s personal information via `a command line tool <https://docs.mattermost.com/administration/command-line-tools.html#platform-user-delete>`_. The command permanently deletes all user information including message contents and files made by the user.
```
Comment (crspeller) [inline source/overview/compliance.rst]: Not true. User's files uploaded are not deleted.

File: source/overview/compliance.rst [hunk 11]
```diff
@@ -1,6 +1,76 @@
...
+
+Security Infrastructure 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost enables organizations to protect their information, and the information of their users and customers, through self-hosted communication infrastructure developed with a high standard of security, including: 
+
+- **Security Features** in Mattermost open source and commercial offerings that enable deployment on your organization’s own infrastructure
+- **Responsible Disclosure Policy** for security researchers around the world to confidentially report suspected vulnerabilities, which can be addressed in updates to Mattermost software 
+- **Security Reviews** conducted by both our own internal security review team and external security researchers
+- **ISO 27001 Standards** which are met to achieve alignment with international security guidelines
```
Comment (crspeller) [inline source/overview/compliance.rst]: Really? We have been audited by them?

File: source/overview/compliance.rst [hunk 12]
```diff
@@ -1,6 +1,76 @@
...
+
+Mattermost adheres to contractual obligations for ensuring the proper management of data through: 
+
+- **GDPR compliant Data Processing Addendum** included with Mattermost’s standard terms
+- **Mattermost Privacy Policy** sharing how data is handled on the online infrastructure controlled by Mattermost, Inc. 
+
+Privacy Measures 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost self-certifies that it adheres to the **E.U.-U.S. Privacy Shield** principles of notice, choice, onward transfer, security, data integrity, access and enforcement for personal data submitted by our customers and partners.
```
Comment (crspeller) [inline source/overview/compliance.rst]: A quick skim of some of the requirements of adhering to this shows that we do not.

Comment (jasonblais) [reply source/overview/compliance.rst]: We're working on getting a doc for this from lawyer today. Will add a link to it.

File: source/overview/compliance.rst (hunk 10 above)
Comment (jasonblais) [reply source/overview/compliance.rst]: Will update

**Comment by jasonblais** [comment]
@crspeller I'm going to merge this EOD. Let me know if you have feedback about my updates or if we're good to go.

**Comment by jasonblais** [comment]
I will send you the legal doc about privacy shield tomorrow when I get it from lawyer.

**Pull request merged**